tests/phpunit/elementor/modules/atomic-widgets/test-atomic-widget-base.php

```diff
@@ -203,37 +195,39 @@ public function test_get_props_schema__is_serializable() {
 		$this->assertJsonStringEqualsJsonString( '{
 			"string_prop": {
 				"type": "string",
-				"constraints": [
-					{ "type": "enum", "value": ["value-a", "value-b"] }
-				],
-				"default": "value-a"
+				"enum": ["value-a", "value-b"],
+				"default": "value-a",
+				"dynamic_categories": [
+					"text"
+				]
 			},
 			"number_prop": {
 				"type": "number",
-				"constraints": [],
-				"default": 123
+				"default": 123,
+				"dynamic_categories": [
+					"number"
+				]
```

Comment:
@Nevoss 

I don't like the fact that we check the "dynamic_categories" here
any idea how tp avoid that?

Reply:
I feel it should live outside of the `Prop_Types` mechanism and be independent